DmitryBarskov/rails-template
# Gather Github Actions template in one file

## Summary
Resolves #37 

Collect all the code related to GitHub actions setup in one file, so that it can be applied using a remote URL.

## Test plan
1. generate a new app with PG, rspec, brakeman, bundler-audit using `rails new myapp --minimal` # TODO
2. apply `github_actions` template: `bin/rails app:template LOCATION=https://raw.githubusercontent.com/DmitryBarskov/rails-template/767291d7949affabede1d4b1ccbbde6188666cac/templates/github_actions.rb`
3. create a repository and push the changes there
4. the github actions workflow should be triggered and probably pass
